.env.example

```
@@ -0,0 +1,8 @@
+TOKEN_SECRET=YOUR_TOKEN_SECRET
+SALT_ROUNDS=SALT_ROUNDS
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The SALT_ROUNDS key should go before the TOKEN_SECRET key

(UnorderedKey)